<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumView</class>
 <widget class="QWidget" name="AlbumView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QLabel" name="albumDisplay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="artist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>204</red>
              <green>144</green>
              <blue>175</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>192</red>
              <green>132</green>
              <blue>161</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>192</red>
              <green>132</green>
              <blue>161</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>204</red>
              <green>144</green>
              <blue>175</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>192</red>
              <green>132</green>
              <blue>161</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>192</red>
              <green>132</green>
              <blue>161</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>艺术家</string>
        </property>
        <property name="text">
         <string>艺术家</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="albumName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>专辑名称</string>
        </property>
        <property name="text">
         <string>专辑名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="albumOps" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="playall">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip">
            <string>从第一个开始播放</string>
           </property>
           <property name="styleSheet">
            <string notr="true">#playall{
background-color: rgb(3, 3, 3);
border-radius:20px;
}
#playall:hover{
background-color:rgb(85, 85, 85);
}
#playall:pressed{
background-color:rgb(0,0,0);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="player-assets.qrc">
             <normaloff>:/mplayer/play_w_fill.svg</normaloff>:/mplayer/play_w_fill.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loveall">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#loveall{
border-radius:20px;
border-color: rgb(0,0,0);
border-style:solid;
border-width:3px;
}
#loveall:hover{
	background-color: rgba(225, 225, 225, 100);
}
#loveall:pressed{
	background-color: rgba(222, 222, 222, 120);
}</string>
           </property>
           <property name="text">
            <string>收藏所有</string>
           </property>
           <property name="icon">
            <iconset resource="player-assets.qrc">
             <normaloff>:/mplayer/heart.svg</normaloff>:/mplayer/heart.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseLocal">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#browseLocal{
border-radius:20px;
border-color: rgb(0,0,0);
border-style:solid;
border-width:3px;
}
#browseLocal:hover{
	background-color: rgba(225, 225, 225, 100);
}
#browseLocal:pressed{
	background-color: rgba(222, 222, 222, 120);
}</string>
           </property>
           <property name="text">
            <string>浏览本地音乐</string>
           </property>
           <property name="icon">
            <iconset resource="player-assets.qrc">
             <normaloff>:/mplayer/folder.svg</normaloff>:/mplayer/folder.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="musicList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget,QListWidget *{
background:transparent;
border:none;
outline:none;
}

QListWidget::item:hover{
border-left:3px solid rgba(255, 200, 255, 100);
}

QListWidget::item:selected{
border-left:3px solid rgb(255, 221, 225);
}

QScrollBar{
border:none;
	background-color: transparent;
width:8px;
}
QScrollBar::handle{
border-radius:4px;
background-color:rgba(62,62,62,160);
}
QScrollBar::add-line{
background:none;
border:none;
}
QScrollBar::sub-line{
background:none;
border:none;
}
QScrollBar::add-page{
background:none;
border:none;
}
QScrollBar::sub-page{
background:none;
border:none;
}
QScrollBar:hover{
	background-color: rgba(234, 234, 234, 120);
}
QScrollBar::handle:hover{
	background-color: rgb(81, 81, 81);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="player-assets.qrc"/>
 </resources>
 <connections/>
</ui>
